File: Julia/8kyu/8kyu-you-can't-code-under-pressure-#1.jl
# You Can't Code Under Pressure #1
# Language: Julia
# Difficulty: 8 kyu

function double_integer(i)
  i * 2
end


# Tests
using FactCheck

facts("Basic tests") do
  context("Should pass fixed tests") do
    @fact double_integer(2) --> 4
    @fact double_integer(4) --> 8
    @fact double_integer(-10) --> -20
    @fact double_integer(0) --> 0
    @fact double_integer(100) --> 200
  end
end

File: Python/8kyu/8kyu-you-can't-code-under-pressure-#1.py
# You Can't Code Under Pressure #1
# Language: Python
# Difficulty: 8 kyu

def double_integer(i):
    return i * 2


# Tests
import codewars_test as test

@test.describe('Tests')
def tests():
    @test.it('Sample Test Case')
    def sample_test_case():
        test.assert_equals(double_integer(2), 4);
        test.assert_equals(double_integer(4), 8);
        test.assert_equals(double_integer(-10), -20);
        test.assert_equals(double_integer(0), 0);
        test.assert_equals(double_integer(100), 200);